addons/test_discuss_full/tests/test_performance.py

```diff
@@ -452,6 +452,7 @@ def _get_init_store_data_result(self):
                     "voice_active_duration": 200,
                     "volumes": [("ADD", [])],
                     "embedded_actions_config_ids": {},
+                    "color_scheme": "light",
```

Comment:
Just a heads up, the issue is still present here, this will still fail with community only.

Reply:
🤦  Indeed...